packages/react-table/src/components/Table/examples/Table.md

```diff
@@ -365,6 +369,159 @@ class SortableTable extends React.Component {
 }
 ```
 
+### Sortable with toolbar
+
+Sorting a table may also be controlled with a toolbar. This is useful for small screen sizes, when the table will responsively shrink into a compact form that displays column headers per row and selecting a column header will not work to sort a table.
+
+```js
+import React from 'react';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  sortable,
+  SortByDirection,
+  wrappable,
+  cellWidth,
+  info
+} from '@patternfly/react-table';
+import {
+  Toolbar,
+  ToolbarItem,
+  ToolbarContent,
+  OptionsMenu,
+  OptionsMenuItemGroup,
+  OptionsMenuItem,
+  OptionsMenuSeparator,
+  OptionsMenuToggle
+} from '@patternfly/react-core';
+import SortAmountDownIcon from '@patternfly/react-icons/dist/esm/icons/sort-amount-down-icon';
+
+class SortableTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      columns: [
+        { title: 'Repositories', transforms: [sortable] },
+        {
+          title: 'Branches',
+          transforms: [
+            info({
+              tooltip: 'More information about branches'
+            }),
+            sortable
+          ]
+        },
+        { title: 'Pull requests', transforms: [sortable] },
+        {
+          title: 'Workspaces',
+          transforms: [sortable]
+        },
+        {
+          title: 'Last commit',
+          transforms: [
+            info({
+              tooltip: 'More information about commits'
+            })
+          ]
+        }
+      ],
+      rows: [
+        ['one', 'two', 'a', 'four', 'five'],
+        ['a', 'two', 'k', 'four', 'five'],
+        ['p', 'two', 'b', 'four', 'five']
+      ],
+      sortBy: {},
+      isSortDropdownOpen: false
+    };
+
+    this.onSort = this.onSort.bind(this);
+    this.onToggle = this.onToggle.bind(this);
+  }
+
+  onSort(_event, index, direction) {
+    const sortedRows = this.state.rows.sort((a, b) => (a[index] < b[index] ? -1 : a[index] > b[index] ? 1 : 0));
+    this.setState({
+      sortBy: {
+        index,
+        direction
+      },
+      rows: direction === SortByDirection.asc ? sortedRows : sortedRows.reverse()
+    });
+  }
+
+  onToggle(isSortDropdownOpen) {
+    this.setState({
+      isSortDropdownOpen
+    });
+  }
+
+  render() {
+    const { columns, rows, sortBy, isSortDropdownOpen } = this.state;
+
+    return (
+      <React.Fragment>
+        <Toolbar id="toolbar">
+          <ToolbarContent>
+            <ToolbarItem>
```

Comment:
Should we hide this toolbar item at wider screen widths? LIke only show it if the table headers are hidden?

Reply:
Actually, Im noticing that this new demo lives under the html-demos for Table. Maybe it should be moved to `React demos` and toggle the control at wider screen widths? and link to the demo from the sortable example where you explain this example is not responsive

Reply:
What do you mean by link? Toolbar has its own docs for the customization of visibility and layout, etc so I'm not sure it's necessary here - the example shows more how to control Table sorting with a custom control

Reply:
The HTML demo is a bit more full-fledged than this example, having a full toolbar and sitting on a page. Maybe we can add that in a follow up?

Reply:
I think that the context may be a little important here. I'm imagining setting it up like this.
There are already sortable table examples - but they are not responsive. So above the sortable table examples we should state that and then link to a Table Demo where we show how to use the sortable table in a whole page, with context and with the responsive solution - like the HTML demos.
Without the context, I think this is a confusing example. What do you think @mcarrano

Reply:
I agree. Cant we just create a react version of this demo that puts everything in context? https://www.patternfly.org/v4/components/table/html-demos/sortable/  Notice that only when the table headers are hidden the sort control is the toolbar appears.

Reply:
I think we could either have two demos - or just implement a composable demo.

but if we have only one demo, we should still link to it from both the legacy and composable examples so that we can still highlight the fact that neither existing examples are responsive.